node/core/backing/src/lib.rs

```diff
@@ -857,7 +858,7 @@ impl util::JobTrait for CandidateBackingJob {
 					&mut tx_from,
 					|tx| RuntimeApiRequest::AvailabilityCores(tx),
 				).await?,
-			)?;
+			).map_err(Error::JoinMultiple)?;
```

Comment:
It would help to also wrap all `request` calls in this `try_join` in `try_runtime_api` to avoid silent exits.

Reply:
72cc8b7